.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     container: golang:1.22
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

Comment:
This is a major version upgrade, which I think is fine but we might want to note it in the log.

Reply:
Good call!  Will add a note in the commit message.